helm/prometheus-rules/templates/alerting-rules/systemd.management-cluster.rules.yml

```diff
@@ -33,3 +33,14 @@ spec:
         severity: page
         team: biscuit
         topic: infrastructure
+    - alert: ManagementClusterRuntimeSystemdUnitFailed
+      annotations:
+        description: '{{`Runtime systemd unit {{ $labels.name }} is failed on {{ $labels.instance }}.`}}'
+        opsrecipe: critical-systemd-unit-failed/
+      expr: node_systemd_unit_state{cluster_type="management_cluster",name=~"docker.service|containerd.service",state!="active"} == 1
+      for: 5m
```

Comment:
5m seems a little low - docker could have PLEG issues which can take some time to resolve.

15m seems reasonable to me.